tools/db_bench_tool.cc

```diff
@@ -4592,6 +4593,12 @@ void VerifyDBFromDB(std::string& truth_db_name) {
         abort();
       }
 
+      if (thread->shared->write_rate_limiter) {
+        thread->shared->write_rate_limiter->Request(
+            key.size() + value_size_, Env::IO_HIGH, nullptr /*stats*/,
+            RateLimiter::OpType::kWrite);
+      }
```

Comment:
Looking at other uses of write_rate_limiter in this file, it seems like more needs to be done. 
For example, if you look at `TimeSeriesWrite`, it seems to be doing: 
```
    if (FLAGS_benchmark_write_rate_limit > 0) {
      write_rate_limiter.reset(
          NewGenericRateLimiter(FLAGS_benchmark_write_rate_limit));
    }
```
Don't you need to do something like this?